### DISCUSSION TITLE
URL change?

### DISCUSSION THREAD

**Discussion by sangxia** [theOehrly/Fast-F1#190] [Q&A]
I'm getting return value None and if I visit for example [here](https://livetiming.formula1.com/static/2022/2022-06-12_Azerbaijan_Grand_Prix/2022-6-10_Practice_2/SessionInfo.json) I get a NoSuchKey error. In fact, this seems to be the case for all 2022 races. I have no problem retrieving 2021 data.

**Comment by sangxia** [comment]
Never mind seems to work now

**Reply by theOehrly** [reply]
Everything seems to be working properly. The url that you posted is incorrect, though. That's why there is an error.
